Wangscape/Options.h

```diff
@@ -17,13 +20,20 @@ class Options
     typedef std::vector<Clique> CliqueList;
 
     std::string filename;
+    std::string directory;
```

Comment:
I'm thinking about extracting "meta data" of options from this class. What do you think?

I've looked over the all changes and they look good, I want to check the structure in detail this evening and we could merge it tomorrow.